packages/html/ds/src/icon/icon.html

```diff
@@ -60,7 +60,8 @@
       {{ getIconStyle(props) | trim }}
       {{ getDisabled(props) | trim }}
       {{ getInline(props) | trim }}
-      {{ getFontSize(props) | trim }}"
+      {{ getFontSize(props) | trim }}
+      {{ props.className | default('') | trim }}"
```

Comment:
@BogdanDanilescu 

I added this change because react `Icon` got the same functionality as well for class names.